package.json

```diff
@@ -26,57 +26,54 @@
       "suppress-exit-code prettier --write"
     ]
   },
-  "resolutions": {
-    "markdown-it": "^12.3.2",
-    "simple-get": "^4.0.1"
```

Comment:
No longer needed because of recent upgrades in transient dependencies

```
├─ prebuild-install@npm:7.0.1
│  └─ simple-get@npm:4.0.1 (via npm:^4.0.0)
│
└─ sharp@npm:0.29.3
   └─ simple-get@npm:4.0.1 (via npm:^4.0.0)
```